mailbox/plugin/quota-search-elasticsearch/src/main/java/org/apache/james/quota/search/elasticsearch/events/ElasticSearchQuotaMailboxListener.java

```diff
@@ -60,7 +60,9 @@ public boolean isHandling(Event event) {
 
     @Override
     public void event(Event event) throws JsonProcessingException {
-        handleEvent(event.getUser(), (QuotaUsageUpdatedEvent) event);
+        if (event instanceof QuotaUsageUpdatedEvent) {
```

Comment:
The change is wrong: the EventBus always call `isHandling()` before calling `event()`, if a test doesn't replicate this behavior, we need to change the test.